e2e/morse_relay_test.go

```diff
@@ -24,26 +25,28 @@ func Test_MorseRelay(t *testing.T) {
 	defer teardownFn()
 
 	tests := []struct {
-		name         string
-		reqMethod    string
-		serviceID    string
-		serviceAlias string
-		relayID      string
-		body         string
+		name      string
+		reqMethod string
+		serviceID protocol.ServiceID
+		reqPath   string
+		relayID   string
+		body      string
 	}{
 		{
-			name:         "should successfully relay eth_chainId for eth (F00C)",
-			reqMethod:    http.MethodPost,
-			serviceAlias: "F00C",
-			relayID:      "1201",
-			body:         `{"jsonrpc": "2.0", "id": "1201", "method": "eth_chainId"}`,
+			name:      "should successfully relay eth_chainId for eth (F00C)",
+			reqMethod: http.MethodPost,
+			serviceID: "F00C",
+			reqPath:   "/v1/abcdef12",
```

Comment:
This should not be made part of the test case, as it is the same for all test cases.

Reply:
Yes but we may want the functionality to run the test for other services, which is why I added it to the test variables.

Reply:
PATH does not differentiate between requests based on this part of the URL at this point. E2E tests need to make this clear by explicitly declaring the value as a single variable/constant used by all the tests.

Reply:
OK for `reqPath` you're right and that makes sense. But for the service ID IMO it's worth keeping as a test far.